crates/hulk/src/webots/keyboard_device.rs

```diff
@@ -29,10 +29,8 @@ impl KeyboardDevice {
             }
         };
 
-        let chest_button_pressed = if let Some(key) = key {
-            key == Keyboard::CONTROL | Keyboard::SHIFT | 'C' as u32
-        } else {
-            received_message.is_some()
+        let chest_button_pressed = received_message.is_some() | {
+            key == Some(Keyboard::CONTROL | Keyboard::SHIFT | 'C' as u32)
         };
```

Comment:
`|` is the bitwise or operation.
In the line below it is used to combine the modifier values with the character to get a keycode, are you sure you want to use that here to combine booleans?

Also, why are there curly braces here?

Reply:
Both mistakes are linked, I didn't use the proper || operator hence rust analyzer complained and wanted curly braces, I didn't think twice as to why, is fixed now